Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -84,9 +91,10 @@ struct CombinatorialKalmanFilterExtensions {
       Delegate<Result<std::pair<typename candidate_container_t::iterator,
                                 typename candidate_container_t::iterator>>(
           candidate_container_t& trackStates, bool&, const Logger&)>;
-  using BranchStopper =
-      Delegate<BranchStopperResult(const CombinatorialKalmanFilterTipState&,
-                                   typename traj_t::TrackStateProxy&)>;
+  using BranchStopper = Delegate<BranchStopperResult(
+      const CombinatorialKalmanFilterTipState&,
+      const typename track_container_t::TrackProxy&,
+      const typename track_container_t::TrackStateProxy&)>;
```

Comment:
It should be - I think it gets only called in case of a new track state which is appended to the end of the track. Either a new measurement or a new hole / material state.